### PR TITLE
Fix Incorrect Tab headings in tab overview after deleting all zim files

### PR DESCRIPTION
**Pull request by iadeelzafar**
### Changes: 
* Add an else if statement to check if the webViewTitle contains the URI
* Set the webViewTitle to "Home"

### CODE REVIEW

File: app/src/main/java/org/kiwix/kiwixmobile/main/TabsAdapter.java [hunk 1]
```diff
@@ -45,10 +45,12 @@
   private final MainActivity activity;
   private TabClickListener listener;
   private int selectedPosition = 0;
+  private Context context;
 
-  TabsAdapter(MainActivity activity, List<KiwixWebView> webViews) {
+  TabsAdapter(MainActivity activity, List<KiwixWebView> webViews, Context context) {
```
Comment (abdulwd) [inline app/src/main/java/org/kiwix/kiwixmobile/main/TabsAdapter.java]: Activity inherits context, so you don't need to add `context` you can use `activity` instead.

Comment (iadeelzafar) [reply app/src/main/java/org/kiwix/kiwixmobile/main/TabsAdapter.java]: @abdulwd yeah, that's better. Done.

Comment (iadeelzafar) [reply app/src/main/java/org/kiwix/kiwixmobile/main/TabsAdapter.java]: Hi @abdulwd , Review this please.

Comment (iadeelzafar) [reply app/src/main/java/org/kiwix/kiwixmobile/main/TabsAdapter.java]: @abdulwd I made the changes. Review this please.

**Pull request merged**